cmd/generate/flag.go

```diff
@@ -15,6 +15,8 @@ const (
 	flagGithubToken   = "github-token"
 	flagInstallation  = "installation"
 
+	flagLocalGenerator = "local-generator"
```

Comment:
How about this?

```
flagDir = "dir"
```

Reply:
I thought about that, but we don't pass the path anywhere, so it anyway works only in the root config path

Reply:
Why not `--dir .` then?